frontend/pages/open-call/[id]/edit.tsx

```diff
@@ -101,9 +101,7 @@ const EditOpenCall: PageComponent<EditOpenCallProps, FormPageLayoutProps> = ({
         onComplete={handleOnComplete}
         initialValues={openCall}
         isLoading={!openCall && isFetchingProject}
-        // The language of the content will be the same as the language of the account, which the
-        // API will automatically set
-        locale={null}
+        locale={openCall?.language}
```

Comment:
For the projects, we use:
```ts
const contentLocale = defaultValues?.language || userAccount?.language;
```

I wonder if we shouldn't fallback to the account's language here as well. What do you think?

Reply:
I think it's a good idea. I'll add the fallback in the open calls form (much how it was implemented for the projects).